### ISSUE TITLE
Warning of the MPC example compilation

### ISSUE THREAD

**Issue by chibi314** [robotology/osqp-eigen#36]
Hi.
When I compile the MPC example, there appears lots of warnings.
The following is one of them.

```
/home/anzai/Library/osqp-eigen/example/src/MPCExample.cpp: In function ‘int main()’:
/home/anzai/Library/osqp-eigen/example/src/MPCExample.cpp:260:48: warning: ‘bool OsqpEigen::Data::setHessianMatrix(const Eigen::SparseMatrix<Scalar>&) [with T = double]’ is deprecated: Use setHessianMatrix(const Eigen::SparseCompressedBase<Derived> &hessianMatrix) instead [-Wdeprecated-declarations]
     if(!solver.data()->setHessianMatrix(hessian)) return 1;
```

What is the best way to cope with this warning?
And what is the best way to convert `Eigen::Dense` matrix to `Eigen::SparseCompressedBase` matrix?

**Comment by xmba15** [comment]
If you just want to get rid of the warnings, maybe you can just apply a pragma flag to temporarily ignore them.
```
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
  // your code
#pragma GCC diagnostic pop
```

**Comment by xmba15** [comment]
Supposed that you are using gcc or clang for as your compiler.

**Comment by GiulioRomualdi** [comment]
Solved thanks to #48
